### ISSUE TITLE
Not recognized mod

### ISSUE THREAD

**Issue by SnosMe** [maxensas/xiletrade#4]
**1 Added Passive Skill is a Jewel Socket**

```
Rarity: Rare
Armageddon Hope
Medium Cluster Jewel
--------
Item Level: 61
--------
Adds 4 Passive Skills (enchant)
1 Added Passive Skill is a Jewel Socket (enchant)
Added Small Passive Skills grant: 10% increased Damage while affected by a Herald (enchant)
--------
1% reduced Mana Reserved (implicit)
--------
Added Small Passive Skills also grant: +3% to Chaos Resistance
Added Small Passive Skills also grant: +3% to Fire Resistance
1 Added Passive Skill is Agent of Destruction
1 Added Passive Skill is Empowered Envoy
--------
Place into an allocated Medium or Large Jewel Socket on the Passive Skill Tree. Added passives do not interact with jewel radiuses. Right click to remove from the Socket.
--------
Corrupted
--------
Note: ~price 0.5 p
```

**Comment by maxensas** [comment]
Thanks for your contribution SnosMe, good remark !
About medium and large clusters, i decided to remove this mod line for a simple reason : medium cluster now always give +1 Jewel Socket as passive and +2 for large, Cluster was updated on Harvest i think ? don't remember exactly.
On standard, you can trade your olds clusters (keep existing mods) and gain another free suffix if you want, i didn't check for corrupted ones.

**Comment by SnosMe** [comment]
Agree, reasonable.

Also found missing **1% reduced Quantity of Items found** on Ventor's Gamble.

```
Rarity: Unique
Ventor's Gamble
Gold Ring
--------
Quality (Resistance Modifiers): +20% (augmented)
--------
Requirements:
Level: 65
--------
Item Level: 81
--------
10% increased Rarity of Items found (implicit)
--------
+35 to maximum Life
1% reduced Quantity of Items found
38% increased Rarity of Items found
+48% to Fire Resistance
+54% to Cold Resistance
-19% to Lightning Resistance
--------
In a blaze of glory,
An anomaly defying all odds
The "unkillable" beast met the divine
And Ventor met his latest trophy.
```

**Comment by maxensas** [comment]
Exactly, will fix that thanks.
Since Hesit league, GGG decided to remove previous "reduces mod" and keep only "increase mod" in filter list, i need to parse this one aswell an put a negative value, thanks 👍

**Comment by maxensas** [comment]
Fixed in 1.3.1
